mednafen/pce_fast/vdc.cpp

```diff
@@ -38,6 +38,7 @@ static uint32 disabled_layer_color;
 
 static bool unlimited_sprites;
 static bool correct_aspect;
```

Comment:
Ugh, I forgot to get rid of this junk... https://github.com/westonlast/beetle-pce-fast-libretro/commit/1a190a626d3c3331fcabd87613f54791dc10bd4d